src/iop/velvia.c

```diff
@@ -104,6 +104,16 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+const char *description(struct dt_iop_module_t *self)
+{
+  return dt_iop_set_description(self, _("resaturate pixels in a weighted manner that gives more weight\n"
```

Comment:
Maybe just:

"resaturate giving more weight to blacks, whites and low-saturation pixels."